Gemfile

```diff
@@ -1,11 +1,8 @@
 source 'https://rubygems.org'
 
-gem 'base_indexer', '>= 0.5.0'
-gem 'discovery-indexer', '>= 1.0.0' # get constituent data from public_xml
+gem 'base_indexer', '>= 1.0.0'
```

Comment:
base_indexer 1.0.0  requires  discovery-indexer ~> 1.0.1,  which in turn requires   stanford-mods  '~>2.1',   so I simplified Gemfile by removing specific version dependencies for  discovery-indexer and stanford-mods